clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -3592,13 +3627,19 @@ public void testIncrementalAlterConfigs()  throws Exception {
                     new ConfigEntry("compression.type", "gzip"),
                     AlterConfigOp.OpType.APPEND);
 
+            AlterConfigOp alterConfigOp3 = new AlterConfigOp(
```

Comment:
This is an intentionally invalid config value for the `CLIENT_METRICS` type?

Reply:
correct - testing for error response like other 2 operations